spec/registrant_spec.rb

```diff
@@ -0,0 +1,39 @@
+require './lib/registrant'
+
+RSpec.describe Registrant do
+  before(:each) do
+    @registrant_1 = Registrant.new('Bruce', 18, true )
+    @registrant_2 = Registrant.new('Penny', 15 )
+  end
+
+  describe "Initialize" do
+    it "exists" do
+      expect(@registrant_1).to be_an_instance_of(Registrant)
+      expect(@registrant_2).to be_an_instance_of(Registrant)
+    end
+  end
+
+  describe "registrant data" do
+    it "checks registrant_1 data" do
+      expect(@registrant_1.name).to eq("Bruce")
+      expect(@registrant_1.age).to eq(18)
+      expect(@registrant_1.permit?).to eq(true)
+      expect(@registrant_1.license_data).to eq({:written=>false, :license=>false, :renewed=>false})
+    end
+
+    it "checks registrant_2 data" do
+      expect(@registrant_2.name).to eq("Penny")
+      expect(@registrant_2.age).to eq(15)
+      expect(@registrant_2.permit?).to eq(false)
```

Comment:
I _would_ test the permit? method twice though since one instance is using the default value and another is passing in that data.